### DISCUSSION TITLE
Add graphs to canvas

### DISCUSSION THREAD

**Discussion by patsims** [bonsai-rx/bonsai#1668] [Q&A] [answered]
Hello all, 
Just like the title says, I would like to add the visualizer of a **RollingGraph** (or any Design.Visualizers) into **CreateCanvas**. I tried to use the **AddImage** but that only works for images. I tried also send the graph to **ConvertToImage**, but no success. 
The only way that I can think might work is using the **ScreenCaptureStream** and then **Crop** around the graph and **AddImage** to the canvas, but it seems very clunky. 
I would be grateful for any clues on for a direct way to add a graph to the canvas. 
Thank you. 
All the best.
P.

**Comment by bruno-f-cruz** [comment, marked as answer]
This is currently not supported as they are completely different libraries. If you REALLY want to have a rasterized version of the plot, currently, the best strategy is probably what you did, using the screengrab.
